src/compat/event_listeners.ts

```diff
@@ -174,22 +185,51 @@ function videoSizeChange() : Observable<unknown> {
   return observableFromEvent(window, "resize");
 }
 
-const isVisible$ = visibilityChange()
-  .pipe(filter((x) => !x)); // emit false when visible
+// emit true when visible and false when hidden
+const visibilityChange$ = visibilityChange().pipe(map((x) => !x));
```

Comment:
So both are called the same way but one is the inverse of the other? That's weird

Reply:
There is also a problem because this is automatically called on script evaluation.

As such, the initial state won't be communicated on subscription.